packages/common/src/bag3d/common/resources/__init__.py

```diff
@@ -36,16 +37,13 @@
 pdal_prod = PDALResource = PDALResource(exe_pdal=os.getenv("EXE_PATH_PDAL")).app
 
 
-db_connection_docker = db_connection.configured(
-    {
-        "port": int(os.getenv("BAG3D_PG_PORT", 5432)),
-        "user": os.getenv("BAG3D_PG_USER"),
-        "password": os.getenv("BAG3D_PG_PASSWORD"),
-        "dbname": os.getenv("BAG3D_PG_DATABASE"),
-        "host": os.getenv("BAG3D_PG_HOST"),
-        # , "sslmode": os.getenv("BAG3D_PG_SSLMODE", "allow"),
-    }
-)
+db_connection = DatabaseResource(
+    host=EnvVar("BAG3D_PG_HOST").get_value(),
+    user=EnvVar("BAG3D_PG_USER").get_value(),
+    password=EnvVar("BAG3D_PG_PASSWORD").get_value(),
+    port=EnvVar("BAG3D_PG_PORT").get_value(),
+    dbname=EnvVar("BAG3D_PG_DATABASE").get_value(),
+).connection
```

Comment:
EnvVar is the os.getenv() equivalent of dagster. Is it ok to use this or shall I switch back to os.getenv? I think the main difference is logging.

Reply:
Yes it's good to use the dagster types, because of their integration with dagster.